dom.bs

```diff
@@ -1014,6 +1019,19 @@ steps:
  <li><p>Return <var>capture</var>, <var>passive</var>, and <var>once</var>.
 </ol>
 
+<p>The <dfn constructor for=EventTarget><code>EventTarget()</code></dfn> constructor, when invoked,
+must run these steps:
+
+<ol>
+ <li>Return a new {{EventTarget}} object, using the default <a>get the parent</a> algorithm, and
```

Comment:
`<li><p>` or maybe inline it in the previous step. It seems that you get a default get the parent and no behavior should maybe be a note as it's just a statement of fact.